package.json

```diff
@@ -1,27 +1,30 @@
 {
-  "name": "react-virtualized",
+  "name": "@automattic/react-virtualized",
   "description": "React components for efficiently rendering large, scrollable lists and tabular data",
   "author": "Brian Vaughn <brian.david.vaughn@gmail.com>",
   "user": "bvaughn",
-  "version": "9.21.2",
-  "homepage": "https://github.com/bvaughn/react-virtualized",
-  "main": "dist/commonjs/index.js",
-  "module": "dist/es/index.js",
-  "jsnext:main": "dist/es/index.js",
+  "version": "9.21.2+automattic",
```

Comment:
if we're scoping it, do we need the funky version?

Reply:
No, it's not necessary. I'll update the package and the changelog, then.